### DISCUSSION TITLE
Visualising a different branch

### DISCUSSION THREAD

**Discussion by pezholio** [githubocto/flat-action#8] [Ideas & Feature Requests]
If I want to use the Flat Viewer for a dataset that doesn't have `master` as the default branch, I get an error. For example:

https://flatgithub.com/fsadata/northern-ireland-biotoxin-results-2020

gives a `Hmm, we couldn't find any files in that repo` error. However, the default branch for this repo is `gh-pages`.

I can get around this for individual data files by specifying the sha, eg:

https://flatgithub.com/fsadata/northern-ireland-biotoxin-results-2020?filename=data%2Fnorthern-ireland-biotoxin-results-021020.csv&sha=afc36cca54d1023a0a59e56ee94c3d9411c9224a

However, the output looks kinda odd:

![image](https://user-images.githubusercontent.com/109774/118808645-a3a9ed00-b8a1-11eb-83f6-1e21a9b11968.png)

**Comment by Wattenberger** [comment]
We currently only check master and main branches for files, but your SHA workaround is great! Just fixed the unnecessary error message so you should see a normal view.
